src/main/java/step2/domain/LottoNumberGenerator.java

```diff
@@ -0,0 +1,37 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumberGenerator {
+
+    public static List<Integer> issueNumbers(int upperbound, int count) {
+        if (count > upperbound) throw new IllegalArgumentException();
+
+        return IntStream.rangeClosed(1, upperbound)
+                .boxed()
+                .collect(toShuffledList())
```

Comment:
```suggestion
                .collect(LottoNumberGenerator::toShuffledList)
```

SupressWarnings 가 없도록 수정하면, 이런식으로 쓸수 있지 않을까 제안해봅니다.

Reply:
이 구조로 만들고 싶었는데 실패했습니다ㅠㅠ 혹시 참고할만한 레퍼런스가 있으면 추천해주실 수 있으신가요?

Reply:
만약 함수형의 활용이 어렵다면, 굳이 욕심내지 않고,
지금은 일단 절차지향적으로 List 자료구조에 for 루프를 통해 일일히 add 한 형식으로 먼저 프로그래밍하는 것은 어떨지 추천드립니다.
다음 사다리타기 미션에서, 이런식으로 표현할 수 있는 법을 엄청 많이 배우고 연습하게 되실겁니다!

이번 미션까지는 테스트 코드와 객체 설계에 더 집중하시면 좋을 것 같습니다.
일단 SupressWarnings 는 좋지 않으니 삭제 부탁드립니다.

Reply:
함수형으로 깔끔하게 짜는건 잘 안되더라구요ㅠㅠ 조언 감사합니다! resolve unchecked warning 커밋에서 SuppressWarning은 걷어냈습니다